editor/src/clj/editor/geom.clj

```diff
@@ -387,6 +387,17 @@
          (= (.y min-p) (.y max-p))
          (= (.z min-p) (.z max-p)))))
 
+(def minimalXY-bounding-box (coords->aabb [0 0 0] [0.0011 0.0011 0]))
```

Comment:
a special "filler" AABB fo empty layers in tilemap
And a function that detects such AABB:

Reply:
Should we also call it `minimalXY-aabb`, like the rest of the predefined aabbs?

Reply:
fixed, thx